network/src/inbound/inbound.rs

```diff
@@ -100,51 +80,59 @@ impl<S: Storage + Send + Sync + 'static> Node<S> {
             (listener_address, listener)
         };
         self.set_local_address(listener_address);
-        info!("Node {:x} listening at {}", self.name, listener_address);
+        info!("Initializing listener for node ({:x})", self.name);
 
         let node = self.clone();
         let listener_handle = task::spawn(async move {
+            info!("Listening for nodes at {}", listener_address);
+
             loop {
                 match listener.accept().await {
                     Ok((stream, remote_address)) => {
                         info!("Got a connection request from {}", remote_address);
 
+                        // Wait a maximum timeout limit for a connection request.
+                        let bootnodes = node.config.bootnodes();
```

Comment:
this assignment can be moved before the body of the `loop`, as that list doesn't change during runtime

Reply:
Good catch, thanks